tools/vpt/vpt_segment.xml

```diff
@@ -0,0 +1,940 @@
+                        <expand macro="sanitize_digits"/>
+                    </param>
+                    <section name="segmentation_properties" title="Segmentation properties">
+                        <param argument="--model" type="select" label="Cellpose model">
+                            <option value="cyto2">cyto2</option>
+                            <option value="nuclei">nuclei</option>
+                            <option value="null">null</option>
+                        </param>
+                        <param argument="--model_dimensions" type="select" label="Model dimensions" help="2D applies the 2D Cellpose model to all z-planes specified in the z_layers list separately and the results are combined into a 3D output. 3D applies the native-3D Cellpose model to all z-planes specified in the z_layers list.">
+                            <option value="2D">2D</option>
+                            <option value="3D">3D</option>
+                        </param>
+                        <param argument="--custom_weights" type="data" format="data" optional="true" label="custom Cellpose weights file"/>
+                        <section name="channel_map" title="Channel map">
+                            <param name="red" type="text" value="" optional="true" label="red"/>
+                            <param name="green" type="text" value="" optional="true" label="green"/>
+                            <param name="blue" type="text" value="" optional="true" label="blue"/>
+                        </section>
+                    </section>
+                    <section name="segmentation_parameters" title="Segmentation parameters">
+                        <param name="nuclear_channel" type="text" value="" optional="true" label="nuclear channel"/>
+                        <param name="entity_fill_channel" type="text" value="" optional="true" label="entity fill channel"/>
+                        <param name="diameter" type="integer" min="0" value="30" label="diameter" help="Default 0 value will let Cellpose to automatically estimate the diameter. Too small diameter leads to fragmented cells and Too big one leads to over-merged cells"/>
+                        <param name="flow_threshold" type="float" min="0" max="1" value="0.4" label="flow threshold" help="Maximum allowed error of the flows for each mask. Increase if cellpose is not returning as many ROIs as you’d expect, and decrease if cellpose is returning too many ill-shaped ROIs"/>
+                        <param name="cellprob_threshold" type="float" min="-6" max="+6" value="0.0" label="cellprobe threshold" help="Decrease this threshold if cellpose is not returning as many ROIs as you’d expect. Similarly, increase this threshold if cellpose is returning too ROIs particularly from dim areas."/>
+                        <param name="minimum_mask_size" type="integer" min="0" value="15" label="Minimum number of pixels per mask" help="Can turn off with -1"/>
```

Comment:
If you can turn off with -1 the min should be -1

The help does not sounds like a real sentence